bin/inbox-start

```diff
@@ -39,6 +39,29 @@ SOCKET_TIMEOUT = 2 * 60
 socket.setdefaulttimeout(SOCKET_TIMEOUT)
 
 
+banner = (
+    "\033[1;95m"
+    r"""
+      _   _       _
+     | \ | |     | |
+     |  \| |_   _| | __ _ ___
+     | . ` | | | | |/ _` / __|
+     | |\  | |_| | | (_| \__ \
+     \_| \_/\__, |_|\__,_|___/
+             __/ |
+            |___/
+    """
+    "\033[0m\033[94m"
+    """
+      S Y N C   E N G I N E
+    """
+    "\033[0m"
+    """
+     Use CTRL-C to stop.
+    """
+)
```

Comment:
maybe this is nitpicky, but to me this is significantly less legible with it broken up like this. There's so many quotations I can't see where strings start and end without looking closely at it. Granted it's a banner, so it's not critical functionality here, but still.

Does it still pass validation if it's a single `r"""` block?

Reply:
It contains terminal escapes to make it colorful and that needs `"\033"`, that would become `"\\033"` with `r""`. I came up with another idea, see if you like it.

We could also just remove color.